src/libs/actions/Report.js

```diff
@@ -792,6 +792,50 @@ Onyx.connect({
     callback: handleReportChanged,
 });
 
+/**
+ * After a reportActions change, we may need to update the last visible reportAction
+ * text, author and date store in the report.
+ *
+ * @param {String} reportID
+ * @param {Object} [optimisticReportActions]
+ * @return {Object} Optimistic and failure onyx updates for the report
+ */
+
+function getReportLastVisibleMessageOnyxUpdatesForReportAfterActionChange(reportID, optimisticReportActions) {
```

Comment:
If we keep thinking I'm sure we'll come up with a longer name 🤣 Can we try to shorten it to `getLastVisibleMessageOnyxUpdatesAfterActionChange`?

Reply:
Thanks for leaving a suggested name, I replaced it.